WordOfTheDay.Tests/UnitTests.cs

```diff
@@ -5,48 +5,40 @@
 using WordOfTheDay.Api.Controllers;
```

Comment:
Can you split unit tests? For example if tests are used for testing WordRepository, the file with tests could be WordRepositoryTests.cs